qemu/tests/openflow_acl_test.py

```diff
@@ -174,9 +175,8 @@ def access_service(access_sys, access_targets, disabled, host_ip,
                     test.fail(err_msg)
                 session.close()
 
-    def get_acl_cmd(protocol, in_port, action, extra_options):
+    def get_acl_cmd(protocol, action, extra_options):
```

Comment:
Could you seperate this into two patches? one is telnetserver for windows, the other one is to modify acl control cmd.

Reply:
ok, I have done https://github.com/autotest/tp-qemu/pull/1480, please help review.